components/home/homepage-whatsapp-cta.tsx

```diff
@@ -0,0 +1,27 @@
+import { PrimaryAnchorButton } from "../ui/button";
+import { WhatsAppIcon } from "../ui/icons";
+
+import { HomePageSection } from "./homepage-section";
+
+export function HomePageWhatsAppCTA() {
+  return (
+    <HomePageSection className="px-4 py-6 space-y-4 text-center">
+      <h2 className="font-semibold">Masih belum dapat info yang kamu cari?</h2>
+      <PrimaryAnchorButton
+        block
+        className="text-white bg-[#25d366] focus:ring-[#25d366]"
+        color="custom"
+        href="https://bit.ly/halowarga"
```

Comment:
```suggestion
        href="https://bit.ly/hotlinewarga"
```

Related to my comment here: https://github.com/kawalcovid19/wargabantuwarga.com/pull/506#issuecomment-889610644

Reply:
We can submit some GTM event tracker on this button/link, so we can track on the GA as well

Reply:
We can do it on seperate task, creating #507 for initial step

Reply:
Currently we already have tracker for all link and button mas. 

So automatically we will track name of the button (or text of hyperlink) as well as the url. 

But I might need to test the current design wether it is still working. 

Case in point:. Info_url and click event. We can see more detailed information on the desktop site of GA. ![Screenshot_20210730-113636_Analytics.jpg](https://user-images.githubusercontent.com/23743497/127600994-c005ce03-d5eb-4993-9283-3d3684c7d067.jpg)